charts/db-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: db-operator
-version: 1.23.0
+version: 1.23.1
```

Comment:
Let's update the minor version, it doesn't seem like a patch to me

Reply:
fixed